plugins/frontend/backstage-plugin-github-pull-requests/src/mocks/handlers.ts

```diff
@@ -18,27 +18,32 @@ import { rest } from 'msw';
 import {
   openPullsRequestMock,
   requestedReviewsMock,
+  requestedReviewsCustomQueryMock,
   repoMock,
   marketingSiteMock,
   yourOpenPullRequests,
+  yourOpenDraftPullRequests,
   closedPullsRequestMock,
   backstagePluginArgoCdMocks,
 } from './mocks';
 
 export const handlers = [
   rest.get('https://api.github.com/search/issues', (req, res, ctx) => {
     const query = req.url.searchParams.get('q');
-    if (query === 'is:open is:pr review-requested:@me archived:false') {
-      return res(ctx.json(requestedReviewsMock));
-    } else if (query === 'is:open is:pr author:@me archived:false') {
-      return res(ctx.json(yourOpenPullRequests));
-    } else if (
-      query ===
-      'state:closed in:title type:pr repo:RoadieHQ/backstage-plugin-argo-cd'
-    ) {
-      return res(ctx.json(closedPullsRequestMock));
+    switch(query) {
```

Comment:
I figured converting this to a switch statement was a bit easier to visually parse rather than adding more `else if` blocks

Reply:
Thank you! It is much more understandable with a quick glance!